### PR TITLE
Moving to long press only to start drag

### LINKED ISSUE #176: Native scrolling on mobile
I upgraded Dovetail to 2.6.2 and reordering groups & cards now works on mobile (yay!). However one thing I noticed immediately is that it's quite difficult to use since you can't scroll. You have to put your finger right on the edge of the screen. Feel free to play around on [dovetailapp.com](https://dovetailapp.com) on your phone to see what I mean, or check out [these screenshots](https://imgur.com/a/fWqMC).

I read through [the note on native scrolling](https://github.com/atlassian/react-beautiful-dnd/tree/v2.6.2#opting-out-of-native-scrolling). I don't really want to introduce drag handles to cards & groups since that increases UI complexity, and I also don't want to reduce the width of the draggable area to less than it already is. 

My suggestion would be introducing a 250ms delay before triggering the reorder. This would let people scroll normally while supporting reorder with a ‘long press’. That's how Trello mobile works and what users have become accustomed to. 

Forgive me if I'm missing some technical aspect. I'm not a ‘real’ developer, just a lowly designer 😂

### PR DESCRIPTION
**Pull request by alexreardon**
Closes #176 

Allows for both native scrolling as well as reordering on the same list

The PR also includes some flow fixes

### CODE REVIEW

**Comment by alexreardon** [comment]
Fixes #176

File: README.md [hunk 1]
```diff
@@ -326,21 +323,9 @@ When a user presses their finger (or other input) on a `Draggable` we are not su
 
 If the user lifts their finger before the timer is finished then we release the event to the browser for it to determine whether to perform the standard tap / click action. This allows you to have a `Draggable` that is both clickable such as a anchor as well as draggable.
 
-#### Opting out of native scrolling
```
Comment (alexreardon) [inline README.md]: 🎉

File: src/view/drag-handle/drag-handle-types.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+// @flow
```
Comment (alexreardon) [inline src/view/drag-handle/drag-handle-types.js]: `// @flow` was missing from the top of the file. Adding it made things actually start validating correctly around the place

**Comment by alexreardon** [comment]
@lukebatchelor would you mind taking a look on your sweet device? 

[mobile link](https://deploy-preview-191--react-beautiful-dnd.netlify.com/iframe.html?selectedKind=single%20vertical%20list&selectedStory=simple%20example&full=0&down=1&left=1&panelRight=0&downPanel=storybook%2Factions%2Factions-panel)

**Comment by lukebatchelor** [comment]
Hmmmm.. should I be able to native scroll when scrolling on the draggables? Not working for me, only when on the background. 

Long press works for dragging though

**Comment by alexreardon** [comment]
Yeah i noticed it is not working for andriod

**Comment by alexreardon** [comment]
@lukebatchelor can you take another look? I have made some modifications 🤞

**Comment by alexreardon** [comment]
This is ready for review @jaredcrowe

**Comment by alexreardon** [comment]
I am now targeting master. This is a nice feature level change and can be enjoyed freely as soon as we are happy with it

File: src/view/draggable/draggable-types.js [hunk 3]
```diff
@@ -39,8 +39,8 @@ export type BaseStyle = {|
   // https://css-tricks.com/snippets/css/remove-gray-highlight-when-tapping-links-in-mobile-safari/
   WebkitTapHighlightColor: 'rgba(0,0,0,0)',
 
-  // Added to avoid the *pull to refresh action* and *anchor focus* on Android Chrome
-  touchAction: 'none',
+  // TODO
```
Comment (jaredcrowe) [inline src/view/draggable/draggable-types.js]: TODO?

Comment (alexreardon) [reply src/view/draggable/draggable-types.js]: 💃 


I'll remove it

Comment (alexreardon) [reply src/view/draggable/draggable-types.js]: done

Comment (alexreardon) [reply src/view/draggable/draggable-types.js]: resolved

**Comment by jaredcrowe** [comment]
lgtm 🚢 🛳 ⛴

**Comment by alexreardon** [comment]
I'll wait until monday. Trying to avoid the friday afternoon 🚢 💥

**Comment by alexreardon** [comment]
But i'll get it into master 🎉

**Pull request merged**